### PR TITLE
Added networking values to sidebar and community screen and Kingfisher

### PR DESCRIPTION
**Pull request by lucyxubroad**
## Overview

Added networking values to sidebar and community screen

## Changes Made

- Added Kingfisher to render profile images
- Added user variable to side bar to render side bar with actual user values
- Added profile images to community screen

## Test Coverage

- Manual simulator testing

## Related PRs or Issues (delete if not applicable)

#87 

## Screenshots (delete if not applicable)

<details>

  <summary>Screen Shot Name</summary>

  
  
Screen Shot 2020-11-24 at 8 01 45 AM
Screen Shot 2020-11-24 at 8 01 47 AM

</details>

### CODE REVIEW

File: CoffeeChat/Controller/HomeViewController.swift [hunk 1]
```diff
@@ -100,14 +106,16 @@ class HomeViewController: UIViewController {
...
     @objc private func profilePressed() {
-        let menu = SideMenuNavigationController(rootViewController: ProfileMenuViewController())
-        let presentationStyle: SideMenuPresentationStyle = .viewSlideOutMenuPartialIn
-        presentationStyle.presentingEndAlpha = 0.85
-        menu.presentationStyle = presentationStyle
-        menu.leftSide = true
-        menu.statusBarEndAlpha = 0
-        menu.menuWidth = view.frame.width * 0.8
-        present(menu, animated: true, completion: nil)
+        if let user = user {
```
Comment (chalo2000) [inline CoffeeChat/Controller/HomeViewController.swift:109]: guard let would probably be cleaner

**Review by chalo2000 — APPROVED** [review]
OG work

File: CoffeeChat/Controller/ProfileMenuViewController.swift [hunk 2]
```diff
@@ -24,18 +25,27 @@ class ProfileMenuViewController: UIViewController {
         MenuOption(image: "settings", text: "Settings")
     ]
     private let profileImageSize = CGSize(width: 120, height: 120)
-
+    private var user: User!
```
Comment (pnor) [inline CoffeeChat/Controller/ProfileMenuViewController.swift:28]: Can make this not a force unwrapped optional by moving `self.user = user` above the `super.init` call

**Review by pnor — APPROVED** [review]
Looks good, didn't know what KingFisher did until this PR lol

File: CoffeeChat/Controller/HomeViewController.swift [hunk 3]
```diff
@@ -64,7 +68,9 @@ class HomeViewController: UIViewController {
 
         NetworkManager.shared.getUser(netId: netId).chained { (response: Response<User>) -> Future<Response<Matching?>> in
```
Comment (danielVebman) [inline CoffeeChat/Controller/HomeViewController.swift:69]: Do you need to explicitly give the closure type?

File: CoffeeChat/Controller/HomeViewController.swift [hunk 4]
```diff
@@ -100,14 +106,16 @@ class HomeViewController: UIViewController {
...
-        present(menu, animated: true, completion: nil)
+        if let user = user {
+            let menu = SideMenuNavigationController(rootViewController: ProfileMenuViewController(user: user))
+            let presentationStyle: SideMenuPresentationStyle = .viewSlideOutMenuPartialIn
+            presentationStyle.presentingEndAlpha = 0.85
+            menu.presentationStyle = presentationStyle
+            menu.leftSide = true
+            menu.statusBarEndAlpha = 0
+            menu.menuWidth = view.frame.width * 0.8
+            present(menu, animated: true, completion: nil)
```
Comment (danielVebman) [inline CoffeeChat/Controller/HomeViewController.swift:117]: I think you can just say `present(menu, animated: true)` but I'm not sure.

File: Podfile [hunk 5]
```diff
@@ -17,7 +17,7 @@ use_frameworks!
     # UI Frameworks
     pod 'IQKeyboardManagerSwift'
     pod 'SnapKit'
-
+    pod 'Kingfisher'
```
Comment (danielVebman) [inline Podfile:20]: Alphabetize

**Review by danielVebman — APPROVED** [review]
Nice job! Just a couple small comments

**Pull request merged**